api/rpc/account/account.proto

```diff
@@ -146,7 +146,8 @@ message SignUpRequest {
 
 message VerificationRequest {
   string name = 1;
-  string code = 2;
+  string password = 2;
```

Comment:
We should never trust what is supplied by a random client, I'm only saying there is no need for our client to receive a plaintext password -- even if it received it over TLS. However we receive it, we would want to follow best practices with respect to hashing with an appropriate salt.